app/templates/Uzivatel/show.latte

```diff
@@ -33,35 +26,61 @@
     <style>
 	.table-nonfluid {
 	    width: auto;
-	 }
+	}
+
+        .nested-payments {
+            width: 100%;
+        }
+        .nested-payments tr > * {
+            width: 50%;
+        }
     </style>
 
     <table class="table table-nonfluid">
-
         {if $master}
-        <tr class="error"><th>Sloučen pod uživatele </th><td><a n:href="Uzivatel:show, id=>$master->id">{$master->nick}</a></td></tr>
+        <tr class="error">
+            <th>Sloučen pod uživatele </th>
+            <td><a n:href="Uzivatel:show, id=>$master->id">{$master->nick}</a></td>
+        </tr>
         <tr><td colspan="2">&nbsp;</td></tr>
         {/if}
 
         {if $slaves}
-        <tr n:foreach="$slaves as $slave" class="error"><th>Sloučený uživatel </th><td><a n:href="Uzivatel:show, id=>$slave">{$slave}</a></td></tr>
+        <tr n:foreach="$slaves as $slave" class="error">
+            <th>Sloučený uživatel </th>
+            <td><a n:href="Uzivatel:show, id=>$slave">{$slave}</a></td>
+        </tr>
         <tr><td colspan="2">&nbsp;</td></tr>
         {/if}
 
-	<tr><th>ID</th><td>{$u->id}</td></tr>
-	<tr><th>Oblast - AP</th><td><a n:href="Ap:list, id => $u->Ap->Oblast->id">{$u->Ap->Oblast->jmeno}</a> - <a n:href="Ap:show, id => $u->Ap->id">{$u->Ap->jmeno}</a></td></tr>
-    <tr><th>Typ členství</th><td>{if $hasCC}platné čestné členství{else}{$u->TypClenstvi->text}{/if}</td></tr>
+	<tr>
+            <th>Oblast - AP</th>
+            <td><a n:href="Ap:list, id => $u->Ap->Oblast->id">{$u->Ap->Oblast->jmeno}</a> - <a n:href="Ap:show, id => $u->Ap->id">{$u->Ap->jmeno}</a></td>
+        </tr>
+        <tr n:if="$u->spolek">
+            <th>Typ členství</th>
+            <td>
+                {if $hasCC}platné čestné členství{else}{$u->TypClenstvi->text}{/if}
+                <br>
+                <a n:href="UzivatelActions:exportPdf id=>$u->id" role="button">Registrační formulář</a>
+                <br>
+                {if $u->regform_downloaded_password_sent==0}
```

Comment:
Tohle má být 0 nebo 1? Co chápu kontext, tak by to mělo být 1, ne?

Reply:
To je divny ale asi to nechme bejt.